library/apache2_site

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+# (c) 2013-2014, Christian Berendt <berendt@b1-systems.de>
+#
+# This site is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: apache2_site
+version_added: 1.6
+short_description: enables/disables a site for the Apache2 webserver
+description:
+   - Enables or disables a specified site for the Apache2 webserver.
+options:
+   name:
+     description:
+        - name of the site to enable/disable
+     required: true
+   state:
+     description:
+        - indicate the desired state of the resource
+     choices: ['enabled', 'disabled']
+     default: enabled
+
+'''
+
+EXAMPLES = '''
+# enables the Apache2 site "default"
+- apache2_site: state=enabled name=default
+
+# disables the Apache2 site "default"
+- apache2_site: state=disabled name=default
+'''
+
+import re
+
+def _disable_site(site):
+    name = site.params['name']
+    a2dissite_binary = site.get_bin_path("a2dissite")
```

Comment:
What happens when a2dissite (and a2ensite) isn't available? Can you have the module handle that a bit gracefully?